WordPress/src/main/java/org/wordpress/android/ui/posts/adapters/PostListAdapter.kt

```diff
@@ -72,6 +96,11 @@ class PostListAdapter(
         }
     }
 
+    fun updateItemLayoutType(updatedItemLayoutType: PostListViewLayoutType) {
+        itemLayoutType = updatedItemLayoutType
```

Comment:
We might want to consider checking whether `updatedItemLayoutType` is actually different than `itemLayoutType`. Wdyt?

Reply:
Sure, no point in updating data if it's the same (though it of course shouldn't really happen since it's a toggle!). Added an early exit just in case though.

Reply:
Ohh, you are right. I believe it was being called more than once before we changed this https://github.com/wordpress-mobile/WordPress-Android/pull/9657#discussion_r278543502.